homeassistant/__main__.py

```diff
@@ -1,6 +1,4 @@
 """Start Home Assistant."""
-from __future__ import print_function
```

Comment:
So the reason for this actually used to be that under Python 2, if you were able to install Home Assistant, we would actually tell you that your Python version was not supported. But I guess Python 2 is dead, so it's fine.